### ISSUE TITLE
Define constant for Affine Transform

### ISSUE THREAD

**Issue by kleingeo** [Project-MONAI/MONAI#6506]
The Affine Transforms allows constant padding, but no way to specify the constant value to use, or at least that method is not documented properly.

**Comment by wyli** [comment]
Pytorch's grid sample API doesn't support this option though https://pytorch.org/docs/stable/generated/torch.nn.functional.grid_sample.html we may enable this with the other resample backends.
